HeterogeneousCore/AlpakaTest/src/ES_AlpakaESTestData.cc

```diff
@@ -0,0 +1,9 @@
+#include "HeterogeneousCore/AlpakaTest/interface/AlpakaESTestData.h"
```

Comment:
whey `ES_AlpakaESTestData.cc` and not simply `AlpakaESTestData.cc` ?

Reply:
To follow naming convention recommended in https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideHowToRegisterESData

Reply:
Ah...

It doesn't see to be widely followed (`TYPELOOKUP_DATA_REG` appears in 63 files named `ED_*` and in 478 files that do not follow the convention).

What's the advantage of `src/ES_AlpakaESTestData.cc` over `src/AlpakaESTestData.cc` ?
Does the former get any special treatment from scram or some other tools ?

Reply:
> What's the advantage of `src/ES_AlpakaESTestData.cc` over `src/AlpakaESTestData.cc` ?
> Does the former get any special treatment from scram or some other tools ?

As far as I know, the file does not get any special treatment from any tools. So I guess the only impact is to tell the file intention for the user.